### PR TITLE
DM-31998: Performance tuning

### PR DESCRIPTION
**Pull request by jonathansick**
This PR resolves performance issues seen in Rubin TV's Roundtable deployment. Because querying all objects in the GCS bucket is fairly slow (>1 second), multiple requests can pile up and easily trigger what is effectively denial-of-service attack. The fixes:

- The auto-refresh JavaScript was set to 0.5 seconds. Here I've backed off the refresh period to 10 seconds to prevent a single client from creating a backlog of requests.
- I've updated the Kubernetes deployment with a replication factor of 5. More replicas mean that more pods can service requests at the same time.
- I noticed that rubintv wasn't logging anything; I've added a simple Timer context manager and a log statement to each request handler to make it easier to monitor requests and response times.

### CODE REVIEW

File: src/rubintv/static/js/refresh.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 
-    function refreshDiv(){
-        $('#refresher').load(window.location.href + " #refresher" , function(){
-           setTimeout(refreshDiv, 500);
-        });
-    }
\ No newline at end of file
+function refreshDiv() {
+  $('#refresher').load(window.location.href + ' #refresher', function () {
+    setTimeout(refreshDiv, 10000);
```
Comment (SimonKrughoff) [inline src/rubintv/static/js/refresh.js:7]: One thing this sets is how fast the 'monitor' channels refresh.  This is important for people waiting for the most recent image to show up.  I think people may notice a 10 second delay.  Do you think it's possible to push this down to `5000` ms, or will that result in a query backlog, do you think?

File: src/rubintv/timer.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import time
+from typing import Any
+
+__all__ = ["Timer"]
+
+
+class Timer:
+    """Time a context.
+
+    Access the duraction after the context exists with the ``seconds``
```
Comment (SimonKrughoff) [inline src/rubintv/timer.py:14]: duraction --> duration

**Review by SimonKrughoff — APPROVED** [review]
Looks great.  Just a couple of minor questions

File: src/rubintv/static/js/refresh.js (hunk 1 above)
Comment (jonathansick) [reply src/rubintv/static/js/refresh.js:7]: 5 seconds could work. Anything faster and I'd recommend re-writing Rubin TV around a more sound architecture.

**Pull request merged**